src/libsync/owncloudpropagator.cpp

```diff
@@ -396,9 +396,12 @@ void OwncloudPropagator::start(const SyncFileItemVector& items)
     }
```

Comment:
FYI I added you on several issues here in github, please mention them in the commit message if you think they are handled by this